packages/databyss-ui/components/Editor/slate/ContentEditable.js

```diff
@@ -111,13 +143,17 @@ const SlateContentEditable = ({
     const _nextText = _nextEditableState.value.document.getNode(activeBlockId)
       .text
     if (_nextText !== _prevText) {
-      onActiveBlockContentChange(_nextText, _nextEditableState)
+      const block = _nextEditableState.value.anchorBlock
+      const jsonBlockValue = { ...block.toJSON(), key: block.key }
+
+      onActiveBlockContentChange(_nextText, _nextEditableState, jsonBlockValue)
```

Comment:
We shouldn't be passing Slate JSONs outside of the ContentEditable component, because it breaks our abstraction. The 3rd argument should be `ranges`, the return value of our `getRangesFromBlock` that we'll create in `components/Editor/slate/markup.js`